tests/Benchmarks/Http/HttpParser.cs

```diff
@@ -32,38 +31,33 @@ public class HttpParser
 
     private static readonly System.Text.Http.Parser.HttpParser s_parser = new System.Text.Http.Parser.HttpParser();
 
+    // new T(); is very expensive because it calls Activator.CreateInstance
+    // so we create it once and keep it in the field (the benchmarks are not mutating it's state so it's fine)
+    private T request = new T(); 
+
     [Benchmark]
-    public void RequestLine()
-    {
-        var request = new Request();
-        s_parser.ParseRequestLine(request, s_plaintextTechEmpowerRequestRos, out _, out _);
-    }
+    public void RequestLine() => s_parser.ParseRequestLine(request, s_plaintextTechEmpowerRequestRos, out _, out _);
 
     [Benchmark]
-    public void Headers()
-    {
-        var request = new Request();
-        s_parser.ParseHeaders(request, s_plaintextTechEmpowerHeadersRos, out _, out _, out _);
-    }
+    public void Headers() => s_parser.ParseHeaders(request, s_plaintextTechEmpowerHeadersRos, out _, out _, out _);
 
     [Benchmark]
     public void FullRequest()
     {
-        var request = new Request();
         s_parser.ParseRequestLine(request, s_plaintextTechEmpowerRequestRos, out var consumed, out _);
         s_parser.ParseHeaders(request, s_plaintextTechEmpowerRequestRos.Slice(consumed), out consumed, out _, out _);
     }
 }
 
-class Request : IHttpHeadersHandler, IHttpRequestLineHandler
+public class Request : IHttpHeadersHandler, IHttpRequestLineHandler
 {
     //public Http.Method Method;
     //public Http.Version Version;
     //public string Path;
     //public string Query;
     //public string Target;
 
-    public Dictionary<string, string> Headers = new Dictionary<string, string>();
+    //public Dictionary<string, string> Headers = new Dictionary<string, string>();
```

Comment:
Was commenting this out necessary for some benchmark related reason or is it just cleanup to be consistent with the rest of the commented out code?

Reply:
I commented it because the corresponding struct type has this field commented too. I wanted to make them more "equal" to make the comparison easier.